### PR TITLE
Update PR Template for Jira

### PR DESCRIPTION
**Pull request by cliffede**
Mantis: N/A
Problem / Feature: N/A
Solution: N/A
Testing: N/A
Concerns: None
Expert Reviewers: JR / BW
Sections to be Reviewed: All
Expected maximum review time: 5 minutes

NOTE: The idea with this entry...
Jira or PR for parallel or older branches: <Enter link to Jira or another PR or delete this if not needed>
...is that it can be left blank if the only other branch(es) needing the fix/feature are newer branches, for example, if the fix is done on the latest release branch, then it will be naturally merged into the mainline (master) at some point and so another Jira or PR is not needed.

### CODE REVIEW

**Review by jeremyrowe — APPROVED** [review]
changes look good - thanks

File: pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Mantis: <Enter link to Mantis>
-Problem / Feature: See Mantis
-Solution: See Mantis
-Testing: See Mantis
+Jira: <Enter link to Jira>
```
Comment (BWhitten) [inline pull_request_template.md]: If we format as per: https://github.com/integrations/jira#link-issues-in-github github will auto insert the link I believe.

**Review by BWhitten — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (BWhitten) [reply pull_request_template.md]: Test jira link:
[ABIP-601]

Comment (cliffede) [reply pull_request_template.md]: Works in fb4_subsytems

**Review by BWhitten — APPROVED** [review]

**Pull request merged**